### PR TITLE
Use `HostMetric` instead of `Values`.

### PR DESCRIPTION
**Pull request by Krout0n**
See this PR after #7, #9 is merged.

### CODE REVIEW

**Comment by ne-sachirou** [comment]
So I think this pull request is still a draft.

File: src/lib.rs [hunk 1]
```diff
@@ -66,26 +53,19 @@ impl Agent {
...
-            type F = Box<dyn Fn() -> Values + Send>;
-            let cpu_metric: F = Box::new(|| {
-                let metrics = Self::get_cpu_metrics().unwrap();
-                // This line is prepared for only solving type-puzzle...
-                // After when all metrics are done moving to src/metrics, these line will get useless.
-                Values(metrics.value)
-            });
+
+            type F = Box<dyn Fn() -> HostMetric + Send>;
+            let cpu_metric: F = Box::new(|| Self::get_cpu_metrics().unwrap());
```
Comment (ne-sachirou) [inline src/lib.rs:62]: ❤️

File: src/metric/disk.rs [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
-        let previous = Disk::get().expect("failed to get disk statistics");
-        std::thread::sleep(interval);
-        let current = Disk::get().expect("failed to get disk statistics");
+type Disks = Vec<Disk>;
+
+// TODO: investigate how many seconds are used in mackerel-agent
+const INTERVAL: Duration = Duration::from_secs(10);
+
+impl From<(Disks, Disks)> for HostMetric {
+    fn from((previous, current): (Disks, Disks)) -> Self {
```
Comment (ne-sachirou) [inline src/metric/disk.rs:12]: This is testable, so it's better to write tests. But it's not necessary in this pull request.

**Review by ne-sachirou — APPROVED** [review]
🙆‍♀️

**Comment by Krout0n** [comment]
Thank you.

**Pull request merged**